### DISCUSSION TITLE
Release 1.10.0

### DISCUSSION THREAD

**Discussion by richshaw2015** [richshaw2015/dino-rss-electron#30] [Bugs]
ChangeLog:

🎉 Support for podcast playing in the background.
🎉 Adjust the default window size.
🎉 Fixed some bugs.

<hr /><em>This discussion was created from the release <a href='https://github.com/richshaw2015/dino-rss-electron/releases/tag/v1.10.0'>Release 1.10.0</a>.</em>

**Comment by 635635** [comment]
Where I can check the subscript list? I only can find the recently updated source that I didn't subscrip. Thanks.

**Reply by richshaw2015** [reply]
switch reading mode to ALL from UNREAD
